Makefile

```diff
@@ -66,6 +66,11 @@ manager: prebuild
 manager-clean:
 	rm -f bin/manager
 
+# Install reloader into a cluster via kubectl kudo init
+.PHONY: reloader
+reloader:
+	go run -ldflags "${LDFLAGS}" cmd/kubectl-kudo/main.go init --disable-manager
```

Comment:
so this is deploying something to your cluster while `make run` just runs the process locally?

Reply:
I would probably call this deploy-reloader then?

Reply:
good call, yeah